doc/TRANSACTION.md

```diff
@@ -10,7 +10,8 @@ The hathor transaction serialized
 
 | Field | Size (bytes) | Description |
 | --- | :---: | --- |
-| `change info` | var | Bytes representing the existence of a change output and the bip32 path of the change address |
+| `version_byte` | 1 | Version byte for the transaction apdu protocol |
+| `change_info` | var | Bytes representing the existence of a change output and the bip32 path of the change address |
```

Comment:
If this is really variable in size, how is the deserialization performed?

Reply:
The structure of the change info is described below, it starts with the length of change outputs and each one has the output index and a bip32 path.

Reply:
I see, thanks